DQM/BeamMonitor/plugins/BuildFile.xml

```diff
@@ -1,4 +1,5 @@
 <use name="DQMServices/Core"/>
+<use name="DataFormats/Scalers"/>
```

Comment:
```suggestion
```

Reply:
not in purpose, I may have missed this change.